documentation/ipi-install/modules/ipi-install-required-data-for-installation.adoc

```diff
@@ -7,8 +7,20 @@
 
 Prior to the installation of the {product-title} cluster, gather the following information from all cluster nodes:
 
-* Out-of-band management IP and credentials
+* Out-of-band management IP
```

Comment:
they will need credentials to fill in the install-config.yaml. Maybe this is the incorrect spot for that?

Reply:
@johnwilkins ?